Source/WTF/wtf/StdLibExtras.h

```diff
@@ -39,6 +40,22 @@
 #include <wtf/GetPtr.h>
 #include <wtf/TypeCasts.h>
 
+// FIXME: Custom implementation not needed once all Linux systems use >libstdc++-10.
+#if defined(__cpp_lib_bit_cast) && __cpp_lib_bit_cast >= 201806L
+#define bit_cast std::bit_cast
```

Comment:
This doesn't need to be a `#define` you could just do `using std::bit_cast` I think